From 6c56c493f919642522e6fbe48c23ee80581fbad3 Mon Sep 17 00:00:00 2001
From: Tim S <stahta01@users.sourceforge.net>
Date: Sat, 26 Dec 2015 16:44:59 -0500
Subject: [PATCH 157/159] - DevPakPlugin_wx30_64.cbp: Added compiler option
 [-include "sdk.h"]. (Thanks stahta01)

And, added search path "sdk/wxscintilla/include". (Thanks stahta01)
---
 src/plugins/contrib/devpak_plugin/DevPakPlugin_wx30_64.cbp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/plugins/contrib/devpak_plugin/DevPakPlugin_wx30_64.cbp b/src/plugins/contrib/devpak_plugin/DevPakPlugin_wx30_64.cbp
index 95afd16..8a16f4e 100644
--- a/src/plugins/contrib/devpak_plugin/DevPakPlugin_wx30_64.cbp
+++ b/src/plugins/contrib/devpak_plugin/DevPakPlugin_wx30_64.cbp
@@ -36,4 +36,5 @@
 					<Add option="-fexceptions" />
 					<Add option="-Winvalid-pch" />
+					<Add option='-include &quot;sdk.h&quot;' />
 					<Add option="-DBUILDING_PLUGIN" />
 					<Add option="-DCB_PRECOMP" />
@@ -46,4 +47,5 @@
 					<Add directory="../../../.objs30_64/include" />
 					<Add directory="../../../include" />
+					<Add directory="../../../sdk/wxscintilla/include" />
 				</Compiler>
 				<Linker>
-- 
2.7.1.windows.2

